pkg/cmd/server/start/start_master.go

```diff
@@ -817,7 +816,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		glog.V(1).Infof("Starting %q", controllerName)
 		started, err := initFn(openshiftControllerContext)
 		if err != nil {
-			glog.Fatalf("Error starting %q", controllerName)
+			glog.Fatalf("Error starting %q (%v)", controllerName, err)
```

Comment:
@deads2k upstream use:

```golang
if err != nil {
  glog.Errorf("Error starting %q", controllerName)
  return err
}
```

https://github.com/kubernetes/kubernetes/blob/1ecb40c7acdd4756ff3b319471c9f85625b607a9/cmd/kube-controller-manager/app/controllermanager.go#L429

Do we want to switch to this model?